### PR TITLE
Updating sv.po

### PR DESCRIPTION
**Pull request by Gapstare**
Resolves: *(direct link to the issue)*

Continuing with the translation to Swedish.

- [ ] I have signed off my commits using `-s` or `Signed-off-by`\* (See: [Contributing § DCO](https://github.com/tenacityteam/tenacity/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))
- [ ] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code\*
- [ ] I made sure the title of the PR reflects the core meaning of the issue you are solving\*
- [ ] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"\*

\* indicates required

### CODE REVIEW

**Comment by nbsp** [comment]
Please follow the [contributing guidelines](https://github.com/tenacityteam/tenacity/tree/master/CONTRIBUTING.md) and the template when opening a PR.

File: .github/workflows/cmake_build.yml [hunk 1]
```diff
@@ -2,7 +2,11 @@ name: CMake Build
 
 on:
   push:
+    branches:
```
Comment (emabrey) [inline .github/workflows/cmake_build.yml]: Please do not change `cmake_build.yml`

Comment (Gapstare) [reply .github/workflows/cmake_build.yml]: > Please follow the [contributing guidelines](https://github.com/tenacityteam/tenacity/tree/master/CONTRIBUTING.md) and the template when opening a PR.

I have to compile the whole thing for translating too?



> Please do not change `cmake_build.yml`

Sorry, I did not know that I did. I did not touch the file manually. I tried to follow the contributing guidelines but perhaps I messed something up in the process.

There is a lot of mentions of Audacity that needs to be changed in one go and it's too tedious to do by hand on some website. That's why I opened this.

Comment (nbsp) [reply .github/workflows/cmake_build.yml]: > > Please follow the [contributing guidelines](https://github.com/tenacityteam/tenacity/tree/master/CONTRIBUTING.md) and the template when opening a PR.
> 
> I have to compile the whole thing for translating too?

If you'd followed the template carefully, you would have learned that no, you don't. But that's not the only box left unchecked.

File: locale/sv.po [hunk 2]
```diff
@@ -2627,9 +2595,9 @@ msgstr "Hitta FFmpeg"
 
 #: src/FFmpeg.cpp
 #, c-format
-msgid "Audacity needs the file '%s' to import and export audio via FFmpeg."
+msgid "Tenacity needs the file '%s' to import and export audio via FFmpeg."
```
Comment (fossdd) [inline locale/sv.po:2598]: You changed the msg-ids of the translations. Please don't to that, the compiler doesn't know which which is. Please also try to compile it by translations. If you change stuff that doesn't effect in any way the compiler, like the README.md you don't have to.

**Review by fossdd — CHANGES_REQUESTED** [review]

**Comment by Gapstare** [comment]
This is too complicated, good luck with the project.

**Comment by n0toose** [comment]
If it's of any help for the future with your open source endeavours, you can do the following:

Before making a commit:
- `git commit -m "hello jello" --signoff`

After making a commit:
- `git commit --amend --signoff`
- or `git rebase HEAD~1 --signoff` (replace 1 with the number of commits you want to sign off in the future)

As far as translations are concerned, we use [Weblate](https://hosted.weblate.org/projects/tenacity/tenacity/) in order to simplify the process. We appreciate your efforts, thanks and best of luck <3

**Comment by Gapstare** [comment]
@panos Thanks for the tip, much appreciated. I have come to the conclusion that I will indeed use Weblate instead and someone that knows what they're doing can fix any hardcoded mentions of 'Audacity'. :slightly_smiling_face:

**Pull request closed without merging**